src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1143,7 +1059,7 @@ def run_dm_activities_with_ior(self, tool, create_dataset=False, pool=None, cont
                 dst_path=posix_path)
 
             # create cont3
-            cont3 = self.create_cont(pool, oclass=self.ior_cmd.dfs_oclass.value)
+            cont3 = self.get_container(pool, oclass=self.ior_cmd.dfs_oclass.value)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'oclass' in method call